projects/packages/forms/src/class-jetpack-forms.php

```diff
@@ -107,4 +107,18 @@ public static function is_mailpoet_enabled() {
 		 */
 		return apply_filters( 'jetpack_forms_mailpoet_enable', true );
 	}
+
+	/**
+	 * Returns true if the Integrations UI should be enabled.
+	 *
+	 * @return boolean
+	 */
+	public static function enable_integrations() {
```

Comment:
`enable_integrations` reads like a command, like "turn integrations on". I'd rename it to something like `should_enable_integrations`.